readme.md

```diff
@@ -71,6 +71,7 @@ From within the [ArcGIS Pro conda environment](http://pro.arcgis.com/en/pro-app/
     - `FORKLIFT_AGS_USERNAME` ArcGIS admin username.
     - `FORKLIFT_AGS_PASSWORD` ArcGIS admin password.
     - `FORKLIFT_AGS_SERVER_HOST` ArcGIS host address eg: `localhost`
+    - `FORKLIFT_POOL_PROCESSES` (optional: defaults to 20) Number of [multiprocessing processes](https://docs.python.org/3/library/multiprocessing.html#using-a-pool-of-workers) to be used to parallelize the starting and stopping of services.
```

Comment:
We can experiment. That's why I put it into an optional environmental variable. I had it set to 20 on my machine and didn't notice a huge performance hit.